apps/sms/test/marionette/conversation_test.js

```diff
@@ -84,37 +84,82 @@ marionette('Conversation Panel Tests', function() {
   });
 
   suite('Forward message', function() {
+    var smsBody = 'this is a sms body';
+    var mmsBody = 'this is a mms body';
     setup(function() {
-      messagesApp.launch();
+      var smsThread = ThreadGenerator.generate({
+        body: smsBody
+      });
+
+      var mmsThread = ThreadGenerator.generate({
+        numberOfMessages: 1,
```

Comment:
yes, but I see you had also this explicitely set in conversion_test.js so I didn't want to have a different style :) And I think it makes it quite readable indeed.